presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -1246,29 +1215,6 @@ private Expression toExpression(Object base, Type type)
         }
     }
 
-    private interface PagePositionContext
```

Comment:
I think you don't need to move this to `Interpreters`. This can just be deleted. The only place that uses this was deleted in https://github.com/prestodb/presto/pull/12505/commits/c2bfbef26aa616dc223f10192f194122d10949f7 so you can just pass `null` where `NoPagePositionContext` is created and delete this whole interface.

Reply:
Good catch. I guess `evaluate` is such a common interface that IntelliJ doesn't grey it out even it's not used